src/sinks/http.rs

```diff
@@ -73,7 +73,7 @@ lazy_static! {
     static ref REQUEST_DEFAULTS: TowerRequestConfig = TowerRequestConfig {
         in_flight_limit: InFlightLimit::Fixed(10),
         timeout_secs: Some(30),
-        rate_limit_num: Some(10),
+        rate_limit_num: Some(u64::max_value()),
```

Comment:
Interestingly this is higher than the value actually allowed in configuration due to https://github.com/timberio/vector/issues/3499

I think we may want to use the max `i64` in the documentation since they cannot actually specify the max `u64` as a value.

Reply:
Oh, that's right. I will fix.